src/common/common.js

```diff
@@ -854,6 +854,19 @@ const common = {
     }
     return false;
   },
+
+  /**
+   * Returns the address ready to show to the user. Applies checksum if needed by the nerworkId.
+   * Only considers Rootstock network
+   * @param {string} address
+   * @param {string} network
+   */
+  toChecksumAddressIfNeeded(address, network) {
```

Comment:
This function should receive the networkId that would be used to apply checksum format